### PR TITLE
1. Added Explore Roadmaps Section

### PR DESCRIPTION
**Pull request by Power-PP**
2. Updated Explore Roadmaps Section in horizontal Listview.
3. Added Mentor of Week Section.
4. Added Mentor of week horizontal scroll listview.
5. Updated bottom navigation bar

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`).
This will ensure a smooth and quick review process. Updating the `pubspec.yaml` and changelogs is not required.

- [x] I read the [Contributor Guide].
- [x] I updated/added relevant comments in code.
- [x] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [x] All existing and new tests are passing.
- [x] The analyzer (`flutter analyze`) does not report any problems on my PR.
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change (please indicate a breaking change in CHANGELOG.md and increment major revision).
- [x] No, this is *not* a breaking change.

[Contributor Guide]: https://github.com/adityathakurxd/university/blob/main/.github/CONTRIBUTION.md

### CODE REVIEW

**Comment by Power-PP** [comment]
pic1

Home Page Preview

**Comment by Power-PP** [comment]
Can I close this pull request now?

File: lib/screens/home/widgets/custom_home_page_widgets.dart [hunk 1]
```diff
@@ -1,69 +1,19 @@
...
-                  ],
-                ),
-              ],
-            ),
-          ),
-        ),
-      ),
+    return Column(
+      children: const <Widget>[
+        ExploreRoadmaps(),
```
Comment (adityathakurxd) [inline lib/screens/home/widgets/custom_home_page_widgets.dart:13]: The `ExploreRoadmaps()` and `Mentors()` widget should instead be in the `university/lib/screens/home/home_screen.dart`.

**Review by adityathakurxd — CHANGES_REQUESTED** [review]
The Explore Course and the related widgets look good in the scroll fashion. 🚀
Great work ✨

I just want to request a few minor changes in the mentor widget:

- [ ] Align the Title Text with the start of Name of Mentor
- [ ] Add a shadow to the Mentor tile as is in the explore course.

File: lib/screens/home/widgets/custom_home_page_widgets.dart (hunk 1 above)
Comment (Power-PP) [reply lib/screens/home/widgets/custom_home_page_widgets.dart:13]: I got those 2 changes but I have a doubt. What this means "The ExploreRoadmaps() and Mentors() widget should instead be in the university/lib/screens/home/home_screen.dart."

Comment (Power-PP) [reply lib/screens/home/widgets/custom_home_page_widgets.dart:13]: I think I got it

**Comment by Power-PP** [comment]
I have applied all the changes you suggested. Can you please check it again

**Comment by Power-PP** [comment]
pic2
New Changes

**Review by adityathakurxd — APPROVED** [review]
Awesome work! @Power-PP 🎉 Looks good to merge.

**Comment by adityathakurxd** [comment]
### Thank you so much for your contribution! @Power-PP 
We'd love to build and ship this project so it can be used by students across universities.

**Please do star the repository** 🌟 and, if possible share your story to LinkedIn and feel free to tag me. (@adityathakurxd)
Thank you again. Have a great journey in Open Source!

**Comment by Power-PP** [comment]
yeah, definitely.
If there is any other issue open please let me know.
Thanks

**Comment by adityathakurxd** [comment]
Sure! Thank you again. 🚀

**Pull request merged**